semeio/jobs/design_kw/design_kw.py

```diff
@@ -61,6 +61,13 @@ def all_matched(line, template_file_name, template):
                     f"but this is probably a Perl file"
                 )
             )
+        elif is_xml(template_file_name, template):
```

Comment:
Some day in the future, this code should be typed. Can you add simple typing to the function you added, is it 
`template_file_name: str, template: List[str]` ?

Reply:
(this comment applies to line 83, not this one ...)

Reply:
OK fixed